x-pack/plugins/security_solution/cypress/screens/security_header.ts

```diff
@@ -5,32 +5,79 @@
  * 2.0.
  */
 
-export const ALERTS = '[data-test-subj="navigation-alerts"]';
+// main links
+export const DASHBOARDS = '[data-test-subj="groupedNavItemLink-dashboards"]';
 
-export const BREADCRUMBS = '[data-test-subj="breadcrumbs"] a';
+export const ALERTS = '[data-test-subj="groupedNavItemLink-alerts"]';
 
-export const CASES = '[data-test-subj="navigation-cases"]';
+export const CASES = '[data-test-subj="groupedNavItemLink-cases"]';
 
-export const HOSTS = '[data-test-subj="navigation-hosts"]';
+export const TIMELINES = '[data-test-subj="groupedNavItemLink-timelines"]';
 
-export const KQL_INPUT = '[data-test-subj="queryInput"]';
+export const EXPLORE = '[data-test-subj="groupedNavItemLink-explore"]';
 
-export const ENDPOINTS = '[data-test-subj="navigation-endpoints"]';
+export const MANAGE = '[data-test-subj="groupedNavItemLink-administration"]';
 
-export const TRUSTED_APPS = '[data-test-subj="navigation-trusted_apps"]';
+// nested links
+export const OVERVIEW = '[data-test-subj="groupedNavPanelLink-overview"]';
 
-export const EVENT_FILTERS = '[data-test-subj="navigation-event_filters"]';
+export const DETECTION_RESPONSE = '[data-test-subj="groupedNavPanelLink-detection_response"]';
 
-export const NETWORK = '[data-test-subj="navigation-network"]';
+export const HOSTS = '[data-test-subj="groupedNavPanelLink-hosts"]';
 
-export const RULES = '[data-test-subj="navigation-rules"]';
+export const ENDPOINTS = '[data-test-subj="groupedNavPanelLink-endpoints"]';
 
-export const EXCEPTIONS = '[data-test-subj="navigation-exceptions"]';
+export const TRUSTED_APPS = '[data-test-subj="groupedNavPanelLink-trusted_apps"]';
 
-export const OVERVIEW = '[data-test-subj="navigation-overview"]';
+export const EVENT_FILTERS = '[data-test-subj="groupedNavPanelLink-event_filters"]';
 
-export const REFRESH_BUTTON = '[data-test-subj="querySubmitButton"]';
+export const NETWORK = '[data-test-subj="groupedNavPanelLink-network"]';
+
+export const USERS = '[data-test-subj="groupedNavPanelLink-users"]';
 
-export const TIMELINES = '[data-test-subj="navigation-timelines"]';
+export const RULES = '[data-test-subj="groupedNavPanelLink-rules"]';
+
+export const EXCEPTIONS = '[data-test-subj="groupedNavPanelLink-exceptions"]';
+
+// other
+export const BREADCRUMBS = '[data-test-subj="breadcrumbs"] a';
+
+export const KQL_INPUT = '[data-test-subj="queryInput"]';
+
+export const REFRESH_BUTTON = '[data-test-subj="querySubmitButton"]';
 
 export const LOADING_INDICATOR = '[data-test-subj="globalLoadingIndicator"]';
+
+// opens the navigation panel for a given nested link
+export const openNavigationPanelFor = (page: string) => {
```

Comment:
nit: This function has to be updated whenever the menu changes. It would be great if we could reuse `getAncestorLinksInfo`. If that is not possible, we also could move the panel logic to every test like this:
```
    openNavigationPanel(EXPLORE);
    navigateFromHeaderTo(OVERVIEW);
    cy.url().should('include', OVERVIEW_URL);
```

Reply:
Yes I tried to use `getAncestorLinksInfo`, but I had a lot of problems, I surrendered when I saw that we can not import anything from other plugins such as `@kbn/cases-plugin`, they make cypress crash. We can not load the app_links config from cypress so I decided to be explicit. But after thinking about it I prefer this way, imagine we have a bug in the links config, it will be detected if we do explicit testing, but if we load the links config to test the links it would probably go undetected.

And you are right about the changes, if some link changes we'll have to come here to update it. But at least we have all the configuration centralized in one place, we will only have to update it here and all tests will change the behavior. That's why I prefer having a single source of truth rather than the `openNavigationPanel` call scattered on many different tests.

Thanks for the review!